site/docs/concepts/architecture.md

```diff
@@ -18,9 +18,8 @@ Modules can provide read/write access or produce implicit copies that serve as l
 Fybrik connects to external services to receive policy engine decisions, available data and credentials. Policies, assets and access credentials to the assets have to be defined before the user can run an application. The current abstraction supports 2 different [connectors](./connectors.md): one for data catalog and one for policy manager. It's designed in an open way so that multiple different catalog and policy frameworks of all kinds of cloud and on-prem systems can be supported. The data steward configures policies in an external policy manager over assets defined in an external data catalog. Dataset credentials are retrieved from Vault by using [Vault API](https://www.vaultproject.io/api). Vault uses a custom secret engine implemented with [HashiCorp Vault plugins system](./vault_plugins.md) to retrieve the credentials from where they are stored (data catalog for example).
 
 Once a developer submits an `FybrikApplication` CRD to Kubernetes the ApplicationController will make sure that all the specs are fulfilled and will make sure that the data is made accessible to the user according to the previously defined policies. The `FybrikApplication` holds metadata about the application such the data assets required by the application, the processing purpose and the method of access the user wishes (protocol e.g. S3 or Arrow flight). 
-It uses this information to check with external systems (4) if access or copy is allowed
-and whether restrictive policies such as masking or hashing have to be applied. It compiles blueprints based on on the policy decisions received via the connectors and chooses the modules (5) which are best fit for the requirements that the user specified regarding the access protocol and availability.
-As data assets may reside in different systems the blueprints are compiled in a `Plotter` CRD (6) that specifies which blueprints have to be executed in which cluster.
+It uses this information to check with external systems (4) if access and/or copy is allowed
+and whether restrictive policies such as masking or hashing have to be applied. It generates a `Plotter` CRD (6) based on on the policy decisions received via the connectors and chooses the modules (5) which are best fit for the requirements that the user specified regarding the access protocol and availability.  The plotter contains the modules required and all the information required to deploy them on the cluster where they will run, as well as the flow of data between the asset and the workload through the chosen modules.  The plotter generates a `Blueprint` CRD (7) per cluster, which contains all the information required to deploy the relevant modules for that cluster.
```

Comment:
CRD --> custom resource (need to change in all the doc as well)

Reply:
Changed in all the site doc pages.